testing/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ endif ()
 install(
     CODE "execute_process( \
     COMMAND ${CMAKE_COMMAND} -E create_symlink \
-    ${ZEEK_CONFIG_BTEST_TOOLS_DIR}/data \
+    ../btest/data \
```

Comment:
This does not fix #3266. Like [Johanna wrote](https://github.com/zeek/zeek/issues/3266#issuecomment-3205933911), the problem is that packaging uses a `DESTDIR` install. The problem is the line just below which does not install into the `DESTDIR`, but the default prefix. A fix needs to be somewhat along the lines of what I posted [here](https://github.com/zeek/zeek/issues/3266#issuecomment-3209454493).

As for relative paths, that a different issue, but maybe worth tackling. I would however keep `ZEEK_CONFIG_BTEST_TOOLS_DIR` since it is used in a number of places and adjusting its value might break things here. I needed we could [make its path relative](https://cmake.org/cmake/help/latest/command/cmake_path.html#relative-path) to `CMAKE_INSTALL_PREFIX`.